### ISSUE TITLE
Object Destructuring throws error: The supplied value [object Object] did not pass the test 'isIdentifier'

### ISSUE THREAD

**Issue by benedusi** [alecclarke/to-typescript#54]
I get the following error if the .js file has object Destructuring: **Command 'toTypeScript()' resulted in an error (Debug Failure. Invalid cast. The supplied value [object Object] did not pass the test 'isIdentifier'.**

A simple js with the follow line will cause the error:
`const {test1, test2} = ''`

**Comment by alecclarke** [comment]
Hi, @benedusi

Sorry for the late response. Can you tell me which version of TypeScript you're using?

**Comment by benedusi** [comment]
Hi, @alecclarke 

I just tested with typescript v3.7 and it is working fine, I was using v3.3.4000 when I got the error, but I also had problems with other extensions and VSCode, which made me reinstall VSCode, so I don't know for sure what fixed it.

Anyway, your extension is working perfectly, thank you.

**Comment by dogancelik** [comment]
```coffee
Core = require './core'
File = require './file'
{ preparePageOptions } = require './util'
```

I've had the same problem, fixed the issue by merging #138 (updates `ts-morph` package).
Thank you for the extension @alecclarke.
